recirq/qaoa/placement.py

```diff
@@ -10,28 +10,39 @@
 import cirq.contrib.routing as ccr
 import cirq_google as cg
 
+# Try importing routing utilities from Cirq. If they don't exist try importing tket for routing.
+# This block of code can be replaced with a simple `from cirq import RouteCQC` once a Cirq
+# release containing routing is pushed and no previous Cirq releases that don't contain routing
+# are supported in ReCirq.
 try:
-    # Set the 'RECIRQ_IMPORT_FAILSAFE' environment variable to treat PyTket as an optional
-    # dependency. We do this for CI testing against the next, pre-release Cirq version.
-    import pytket
-    import pytket.extensions.cirq
-    from pytket.circuit import Node, Qubit
-    from pytket.passes import SequencePass, RoutingPass, PlacementPass
-    from pytket.predicates import CompilationUnit, ConnectivityPredicate
+    from cirq import RouteCQC
+    pytket = NotImplemented
```

Comment:
It's a little messy right now. I am declaring `pytket = NotImplemented` whenever Cirq's routing utilities exist to simplify the imports in `placement_test.py`. To remove this line I'd have to add another try-except block on the test file and change the `pytest.pskipif` code. Any opinions on which is better?

Reply:
I think the second option is cleaner so I pushed a new commit for it.